### ISSUE TITLE
Command based multi voice generation does not take effect

### ISSUE THREAD

**Issue by newstargo** [SWivid/F5-TTS#365]
python ./f5_tts/infer/infer_cli.py -c ./f5_tts/infer/examples/multi/story.toml
After generation, only the tones defined by main cannot be effective for town and country tones. How to solve this problem？

**Comment by SWivid** [comment]
how is the command link output, e.g. provide a screenshot
how is the generated sample
what is the toml and txt file content you use

need more info to figure out

**Comment by newstargo** [comment]
(share_venv) E:\F5-TTS\src>python ./f5_tts/infer/infer_cli.py -c ./f5_tts/infer/examples/multi/story.toml
--Load vocos from local path /models/charactr-vocos-mel-24khz
--Using F5-TTS...
--vocab :  E:\F5-TTS\src\f5_tts\infer\examples\vocab.txt
--tokenizer :  custom
--model :  \models\F5TTS_Base\model_1200000.safetensors
--Whisper-v3-turbo 从audio中提取参考词...
E:\share_venv\lib\site-packages\transformers\models\whisper\generation_whisper.py:496: FutureWarning: The input name `inputs` is deprecated. Please make sure to use `input_features` instead.
  warnings.warn(
You have passed task=transcribe, but also have set `forced_decoder_ids` to [[1, None], [2, 50360]] which creates a conflict. `forced_decoder_ids` will be ignored in favor of task=transcribe.
Passing a tuple of `past_key_values` is deprecated and will be removed in Transformers v4.43.0. You should pass an instance of `EncoderDecoderCache` instead, e.g. `past_key_values=EncoderDecoderCache.from_legacy_cache(past_key_values)`.
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Finished transcription
ref_t:Six spoons of fresh snow peas, five thick slabs of blue cheese and maybe a snack for her brother Bob.
gen_text 0 A Town Mouse and a Country Mouse were acquaintances, and the Country Mouse one day invited his friend to come and see him at his home in the fields. The Town Mouse came, and they sat down to a dinner of barleycorns and roots,
gen_text 1 the latter of which had a distinctly earthy flavour. The fare was not much to the taste of the guest, and presently he broke out with [town] “My poor dear friend, you live here no better than the ants. Now, you should just see how I fare!
gen_text 2 My larder is a regular horn of plenty. You must come and stay with me, and I promise you you shall live on the fat of the land.” [main] So when he returned to town he took the Country Mouse with him,
gen_text 3 and showed him into a larder containing flour and oatmeal and figs and honey and dates. The Country Mouse had never seen anything like it, and sat down to enjoy the luxuries his friend provided: but before they had well begun,
gen_text 4 the door of the larder opened and someone came in. The two Mice scampered off and hid themselves in a narrow and exceedingly uncomfortable hole. Presently, when all was quiet, they ventured out again; but someone else came in, and off they scuttled again.
gen_text 5 This was too much for the visitor. [country] “Goodbye,” [main] said he, [country] “I’m off. You live in the lap of luxury, I can see, but you are surrounded by dangers; whereas at home I can enjoy my simple dinner of roots and corn in peace.”
--Generating audio in 6 batches...
  0%|                                                                                            | 0/6 [00:00<?, ?it/s]Building prefix dict from the default dictionary ...
Loading model from cache R:\Temp\jieba.cache
Loading model cost 0.971 seconds.
Prefix dict has been built successfully.
E:\share_venv\lib\site-packages\torch\functional.py:650: UserWarning: ComplexHalf support is experimental and many operators don't support it yet. (Triggered internally at ..\aten\src\ATen\EmptyTensor.cpp:32.)
  return _VF.stft(input, n_fft, hop_length, win_length, window,  # type: ignore[attr-defined]
100%|████████████████████████████████████████████████████████████████████████████████████| 6/6 [00:20<00:00,  3.37s/it]
tests\infer_cli_out.wav

**Comment by newstargo** [comment]
story.toml
# F5-TTS | E2-TTS
model = "F5-TTS"
ref_audio = "E:/F5-TTS/src/f5_tts/infer/examples/multi/main.flac"
# If an empty "", transcribes the reference audio automatically.
ref_text = ""
gen_text = ""
# File with text to generate. Ignores the text above.
gen_file = "E:/F5-TTS/src/f5_tts/infer/examples/multi/story.txt"
remove_silence = true
output_dir = "tests"

[voices.town]
ref_audio = "E:/F5-TTS/src/f5_tts/infer/examples/multi/town.flac"
ref_text = ""

[voices.country]
ref_audio = "E:/F5-TTS/src/f5_tts/infer/examples/multi/country.flac"
ref_text = ""

**Comment by SWivid** [comment]
@newstargo seems an issue that you haven't proper install the pkg
need to follow readme, e.g. it seems you want to do the generation locally, so (after you have set up conda env and proper install torch stuffs) run:
```
git clone https://github.com/SWivid/F5-TTS.git
cd F5-TTS
pip install -e .
```
then
```
f5-tts_infer-cli -c src/f5_tts/infer/examples/multi/story.toml
```

**Comment by SWivid** [comment]
will close this issue, feel free to open if further questions
